src/main.js

```diff
@@ -333,9 +339,8 @@ window.unfollow = async () => {
     }
 
     isActiveProcess = false;
-    elResultsContainer.innerHTML += `<hr /><div style='padding:1rem;font-size:1.25em;color:#56d756;'>All DONE!</div><hr />
-    <button class='iu_main-btn' style='position:absolute;left:50%;top:50%;transform:translate(-50%,-50%);font-size:2em;cursor:pointer;height:160px;width:160px;border-radius:50%;background:transparent;color:currentColor;border:1px solid currentColor;'>RERUN</button>`;
-    getElementByClass('.iu_main-btn').addEventListener('click', () => run(true));
+    elResultsContainer.innerHTML += `<hr /><button class='ui-return-btn'style='padding:1rem;font-size:1.25em;color:#56d756;background:transparent;color:currentColor;'>All DONE! click here to return</button><hr />`;
```

Comment:
No need for `color:currentColor`, there's already a color value on it :)
Also, please add `border:none`, since it now has a black border to it.